tests/test_helpers/test_profiling.py

```diff
@@ -93,21 +109,17 @@ def output_function(report):
         nonlocal result
         result = report
 
-    result = ""
-
-    p = Profiling([Message], 1, output_function=output_function)
+    result, types_to_track = "", [Message]
+    p = Profiling(types_to_track, 1, output_function=output_function)
     p.start()
+    wait_for_condition(lambda: p.is_running, timeout=TIMEOUT)
 
-    wait_for_condition(lambda: p.is_running, timeout=20)
-    m = Message()
     try:
-        wait_for_condition(lambda: result, timeout=20)
-
+        wait_for_condition(lambda: result, timeout=TIMEOUT)
         assert "Profiling details" in result
     finally:
         p.stop()
-        p.wait_completed(sync=True, timeout=20)
-    del m
```

Comment:
We don't instantiate a Message now. Shouldnt we be calling create_dummies?

Reply:
this was not used in the assertion